api/SetupHelper.cs

```diff
@@ -15,6 +16,9 @@ public static void AddServicesAndClients(this IServiceCollection services)
         services.AddScoped<AccountService, AccountService>();
         services.AddScoped<AccountClient, AccountClient>();
 
+        services.AddScoped<TeamselectionService, TeamselectionService>();
```

Comment:
Er bestaat code waarmee je kan zeggen vind alles van een bepaald type en add.
Wss willen we dan een clientinterface/serviceinterface oid hebben en dan kan je TeamselectionClient : BaseClient doen en dat doet de setupHelper het toevoegen automatisch.

Voor nu kan je toevoegen TODO automatically find Clients/services